posthog/api/dashboards/dashboard_templates.py

```diff
@@ -84,7 +84,16 @@ def repository(self, request: request.Request, **kwargs) -> response.Response:
     def _load_repository_listing() -> List[Dict]:
         url = "https://raw.githubusercontent.com/PostHog/templates-repository/main/dashboards/dashboards.json"
         loaded_templates: List[Dict] = json.loads(requests.get(url).text)
-        return loaded_templates
+        return [
+            # The OG template is hard-coded, not served by the repository, so added in to the template listing here
```

Comment:
Why keep this hard-coded instead of adding to the repository?

Reply:
We use it in tests and in initial team setup. So ran into a chicken and egg problem of having to load templates to do anything.... but that's in my brain not the code